catlas/validation.py

```diff
@@ -0,0 +1,203 @@
+from pymatgen.core import Element
+from cerberus import Validator
+from pathlib import Path
+import yaml
+import os
+
```

Comment:
probably rename this to be more intuitive. Validation is very broad. Probably `input_yaml_validation.py`?